cli/compose/convert/service.go

```diff
@@ -36,9 +37,19 @@ func Services(
 		if err != nil {
 			return nil, errors.Wrapf(err, "service %s", service.Name)
 		}
-		configs, err := convertServiceConfigObjs(client, namespace, service.Configs, config.Configs)
-		if err != nil {
-			return nil, errors.Wrapf(err, "service %s", service.Name)
+		var configs []*swarm.ConfigReference
+		if len(service.Configs) > 0 {
```

Comment:
I think that method is a lot more reliable. 

I don't understand why this would be a special case. We added secrets before and didn't have this problem.

We should just error if API version is less than the minimum version required for the Compose file version.